src/components/ReceiveInfo.tsx

```diff
@@ -140,7 +140,7 @@ export const ReceiveInfo = () => {
                     maxWidth: '20rem',
                   }}
                 >
-                  <ApeAvatar user={tokenGift.sender} />
+                  <Avatar path={tokenGift.sender} />
```

Comment:
`sender` here isn't an avatar path

Reply:
I don't know the correct path for the token gift sender
can u tell me?

Reply:
it should be `tokenGift.sender.avatar` since `sender` is a User